workshop/SCHEDULE.md

```diff
@@ -3,17 +3,53 @@ title: Workshop Schedule
 nav_title: Schedule
 ---
 
-<!--See an example from a past remote workshop here: https://github.com/AlexsLemonade/2023-may-training/blob/main/workshop/SCHEDULE.md -->
-<!--See an example from a past in-person workshop here: https://github.com/AlexsLemonade/2023-june-training/blob/main/workshop/SCHEDULE.md -->
+The planned schedule for the June 2024 Data Lab Virtual Training Workshop "Introduction to Single-Cell RNA-Seq" appears below.
 
 
-| Time        | Topic                                          |
-|-------------|------------------------------------------------|
-| **Day 1**   | **Date** <br> <!--uncomment and add link: [Module]()-->                      |
-| 12:00 PM    | Welcome, Introductions and Getting Started     <br>[Welcome Slides (PDF)](../slides/Workshop_Introduction.pdf)|
-| 5:00        | End             |
-| **Day 2**   | **Date**  |
-| **Day 3**   | **Date**  |
-| **Day 4**   | **Date**  |
-| **Day 5**   | **Date**  |
-| 5:00        | Adjourn   |
+*Note: All times are [EDT (UTC−04:00)](https://www.timeanddate.com/time/zones/edt).*
+
+
+| Time        | Topic                             | Location |
+|-------------|--------------------------------------------|----------------|
+| **Day 1**   | **2024-06-10** <br> _**Introduction to R and the Tidyverse**_                 |
+| 12:00 PM    | Welcome, Introductions and Getting Started <br> [Workshop introduction slides (PDF)]()  | Zoom: [Main Session](../software-setup/zoom-procedures.md) |
```

Comment:
For the blank links, let's at least make them easy to find: Using a `#` here also gives the advantage of being an anchor link that doesn't work. The current version will link to the parent directory (at least here that is what it does), which we don't really want. 
```suggestion
| 12:00 PM    | Welcome, Introductions and Getting Started <br> [Workshop introduction slides (PDF)](#STUB)  | Zoom: [Main Session](../software-setup/zoom-procedures.md) |
```